css/styles.css

```diff
@@ -440,3 +472,183 @@ textarea {
   letter-spacing: 0.03em;
   color: #fff;
 }
+
+@media (max-width: 780px) {
+  .menu_desk {
+    display: none;
+  }
+}
+
+@media (min-width: 780px) {
+  header {
+    position: fixed;
+    display: flex;
+    width: 100%;
+  }
```

Comment:
- [ ] Hey, This is an amazing work you've done here. Please, kindly compare your header with the Figma and fix the issue with the background color which is supposed to dominate the background of the body of your page below:

:point_down:  **YOUR DESIGN**


![space](https://user-images.githubusercontent.com/59397159/165687891-5885ba2c-4b90-4592-aded-c7fdc924b0c1.png)

:point_down:  **FIGMA DESIGN**

![REDO](https://user-images.githubusercontent.com/59397159/165688199-05125904-193f-4a68-b702-0ebb471e2201.png)